src/scss/grommet-core/_objects.form-field.scss

```diff
@@ -78,13 +78,39 @@ $form-vertical-padding: quarter($inuit-base-spacing-unit);
 
     > select {
       display: block;
-      background-position: center right $form-horizontal-padding;
+      // TODO figure out how to make this consistent with intuit and not have magic numbers
+      background-position: center right #{$form-horizontal-padding - 6};
+      padding-left: #{$form-horizontal-padding - 2};
```

Comment:
The - 2 doesn't look correct on Chrome or Safari for a Mac. I didn't see an obvious solution to have it look correct in all cases though. Can you add a comment indicating this?